ACPs/77-reinventing-subnets/README.md

```diff
@@ -38,31 +38,39 @@ With no Elastic Subnets live on Mainnet, it is clear that Permissionless Subnets
 
 Subnet Validators will only be required to sync the P-chain (not X/C-Chain) to track any validator set changes in their Subnet and to support Cross-Subnet communication via Avalanche Warp Messaging (see "Primary Network Partial Sync" mode introduced in [Cortina 8](https://github.com/ava-labs/avalanchego/releases/tag/v1.10.8)). The lower resource requirement in this "minimal mode" will provide Subnets with greater flexibility of validation hardware requirements as operators are not required to reserve any resources for C-Chain/X-Chain operation. Since Subnet Validators are no longer required to validate or sync the Primary Network, the 2000 $AVAX requirement can be removed. To meter the number of Subnet validators on the network, a significantly lower, continuous fee is introduced later in this ACP.
 
+#### Bootstrapping Subnet Nodes
+
+Bootstrapping a node/validator is the process of securely recreating the latest state of the chain locally. At the end of this process, the local state of a node/validator must be in sync with the local state of other virtuous nodes/validators. The node/validator can then verify new incoming transactions and reach consensus with other nodes/validators.
+
+To start this process, a node/validator must connect to a _large enough_ majority of validators from the _up-to-date_ validator set. For standalone networks like the Avalanche Primary Network, discovering the up-to-date validator set is a chicken-and-egg problem. Until completing the bootstrapping process, it is impossible to determine whether or not the validator set is stale. Node clients typically hardcode a set of trusted bootstrappers that are already on the network to enable safe bootstrapping. AvalancheGo's list can be found [here](https://github.com/ava-labs/avalanchego/blob/master/genesis/bootstrappers.json).
```

Comment:
> To start this process, a node/validator must connect to a _large enough_ majority of validators from the _up-to-date_ validator set

I'm sure there is a link you can throw in here talking about this sort of attack (where you sync to the wrong chain).

> it is impossible to determine whether or not the validator set is stale

I would recommend `it is impossible to determine the validator set`

> Node clients typically hardcode a set of trusted bootstrappers that are already on the network to enable safe bootstrapping.

This somewhat conflicts with your first sentence ("must connect to a _large enough_ majority of validators from the _up-to-date_ validator set.") -> the bootstrappers typically aren't anywhere close to the majority.